src/mono/mono/mini/aot-runtime.c

```diff
@@ -5049,24 +5068,63 @@ mono_aot_get_method (MonoMethod *method, MonoError *error)
 			method_index = find_aot_method (shared, &amodule);
 			if (method_index != 0xffffff)
 				method = shared;
+			if (method_index == 0xffffff && !mono_method_metadata_has_header (method)) {
```

Comment:
Is it true that method without header will always be extern method with unsafe accessor attribute?